.circleci/config.yml

```diff
@@ -167,14 +191,14 @@ jobs:
       - run:
           name: Build XCFramework
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [ "${CIRCLE_BRANCH}" == "staging" ] || [ "${CIRCLE_BRANCH}" == "dev" ]; then
               make archive
             fi
       # verify test app builds
       - run:
           name: Build Test App
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [ "${CIRCLE_BRANCH}" == "staging" ] || [ "${CIRCLE_BRANCH}" == "dev" ]; then
```

Comment:
Can you update to run on "main" as well? There can be cases where we merge directly to main so this workflow should get run again.